projects/libs/core-library-typescript/src/consensus/stub-consensus.ts

```diff
@@ -73,20 +73,25 @@ export class StubConsensus extends BaseConsensus {
       logger.debug(`Stub consensus leader heartbeat - my last block is ${JSON.stringify(myLastCommittedBlock)}`);
 
       // check if all other nodes are synchronized with me
-      let areAllOtherNodesSyncedWithMe = true;
-      if (this.lastBlockHeightByNodeName.size != this.config.clusterSize - 1) {
+      let areEnoughNodesSyncedWithMe = true;
```

Comment:
I think it would be a bit easier for us to maintain this code if we rewrote it using `filter((k, v) => v <= lastBlockSize)` and then check the size.